content/tutorials/quickstart/create-accounts-send-xrp.md

```diff
@@ -539,18 +524,24 @@ Create a standard HTML form to send transactions and requests, then display the
 <html>
   <head>
     <title>Token Test Harness</title>
-    <script src='https://unpkg.com/xrpl@2.1.1'></script>
+    <script src='https://unpkg.com/xrpl@2.2.3'></script>
     <script src='ripplex1-send-xrp.js'></script>
     <script>
       if (typeof module !== "undefined") {
         const xrpl = require('xrpl')
       }
+
     </script>
   </head>
+  
+<!-- ************************************************************** -->
+<!-- ********************** The Form ****************************** -->
+<!-- ************************************************************** -->
+
   <body>
     <h1>Token Test Harness</h1>
     <form id="theForm">
-      Choose your network:  
+      Choose your ledger instance:  
```

Comment:
I suggest rolling this change back. The term "network" is more consistent with the usage across our other docs, and "ledger instance" is likely to be interpreted to mean an instance of the ledger data type.

Reply:
Will do in the next release.